front/app/containers/Admin/settings/customize/index.tsx

```diff
@@ -276,9 +303,10 @@ class SettingsCustomizeTab extends PureComponent<
             titleError={titleError}
             subtitleError={subtitleError}
             latestAppConfigStyleSettings={latestAppConfigStyleSettings}
-            latestAppConfigCoreSettings={latestAppConfigCoreSettings}
+            latestAppConfigSettings={latestAppConfigSettings}
             setParentState={setState}
             getSetting={getSetting}
+            handleSettingOnChange={this.handleSettingOnChange}
```

Comment:
In the refactor I decided to handle the state updates in the parent (`customize/index`) triggered from child components (like `<Header>`) through the `setParentState` prop, so that the specific logic to update the state is divided among child components, instead of living all in the parent. The `this.handleSettingOnChange` method kind of deviates from that pattern. I would personally put that in `Header` as well, but if you have good reasons for not doing it like this, that's fine too. There are many other things that can be improved in this part of the settings anyway, so maybe better to include as part of a bigger refactor.

Reply:
Sure, agree! But I'm not going to do it now (short on time). If you want, you can make a separate PR for it after this is released.